### ISSUE TITLE
OsmAnd giving frequent and consecutive warnings for closely spaced speed bumps

### ISSUE THREAD

**Issue by jol70** [osmandapp/OsmAnd#17723]
## Description
L exemple ce sont les passage piéton, la remarque est vraie aussi pour les rond points ou les changements de directions. Bien sûr on peut supprimer l information, toutefois si on l utilisé c est que c'est utile. Les gps type waze ou maps on limite la répétions à environ 3 fois et c est suffisant. 

### Expected result
Que l information sonore soit limitée en nombre de rappel et alors que le véhicule est à une distance fonction de sa vitesse et de son poids. Premiere alerte très eloignée  si la vitesse et le poids du véhicule sont important ou fixe à 250m hors ville et 50m en ville  ou réglable Deuxième alerte 20m Troisieme alerte 5m. 
Merci 
**Environment**
OsmAnd Version:
Android/iOS version:
Device model:

**Comment by vshcherb** [comment]
We need to translate issue to english so everybody could follow

**Comment by letypequividelespoubelles** [comment]
I can do the translation, but even the French is not very understandable. 
@jol70 combien de notifications tu as maintenant ? Et merci d'indiquer la version d'osmand, etc
Translated and resumed:
Tok much sound notification for pedestrian crossing, direction change, etc. Three notifications should be enough, first one, far from the poi, at a distance deoemding on the vehicle and city ) outside city, a second one 20m before, and a third one 5m before.

**Comment by yuriiurshuliak** [comment]
Reproducing involves OsmAnd giving frequent and consecutive warnings for closely spaced speed bumps, resulting in an excessive number of audible crosswalk alerts. In contrast, other navigation apps handle this situation differently by providing a more concise instruction like "attention, several (few) bumps ahead."

OsmAnd~ 4.6.0#497m, released: 2023-07-28
OsmAnd 4.4.10

<table style="width:100%">
  </tr>
  <tr>
    <th>iOS</th>
    <th>Android</th>
  </tr>
  <tr>
    <td> <video source src="https://github.com/osmandapp/OsmAnd/assets/127092082/1a27129e-2ab4-402a-94dd-3516baca7930"  width="300" height="auto"/>   </td>
    <td><video source src="https://github.com/osmandapp/OsmAnd/assets/127092082/ef916964-cca3-403d-b7bf-d4d748795b3e"  width="300" height="auto"/>   </td>
  </tr>
</table>

**Comment by ivanPyrohivskyi** [comment]
iOS needed

**Comment by yuriiurshuliak** [comment]
The bug has been resolved. During testing with the latest nightly build of OsmAnd, we did not observe frequent and consistent warnings regarding nearby speed bumps. Currently, when multiple speed bumps are close to each other, the first one triggers a sound warning, while subsequent ones are only indicated by a warning sign in the bottom left corner of the screen. 

A fix in iOS is expected in version 4.6.

https://github.com/osmandapp/OsmAnd/assets/127092082/8879fc16-be95-493b-a215-9a78e12c5edf



OsmAnd~ 4.6.0#818m, released: 2023-09-29
OsmAnd 4.5.2.2
